src/basic/terminal-util.h

```diff
@@ -111,6 +111,9 @@ typedef enum ColorMode {
         /* Only 256 colors. */
         COLOR_256 = 256,
 
+        /* Only 16,777,216 colors. */
+        COLOR_24BIT = 16777216,
```

Comment:
I know that 16777216 was requested here, but this is not useful. Instead, drop the value assignments for COLOR_OFF, COLOR_ON, COLOR_16, COLOR_256 and COLOR_24BIT so that the compiler can assign 0, 1, 2, 3, 4 automatically.

Also "only 16 mil colors" is not a useful comment. It was copied from the comment above, which was copied from the comment about 16 colors, where it made sense. Please adjust this also.

Reply:
ok sure.

Reply:
> I know that 16777216 was requested here, but this is not useful. Instead, drop the value assignments for COLOR_OFF, COLOR_ON, COLOR_16, COLOR_256 and COLOR_24BIT so that the compiler can assign 0, 1, 2, 3, 4 automatically.

Dropped for all mentioned.